config/processEimage.py

```diff
@@ -0,0 +1 @@
+config.calibrate.photoCal.doSelectUnresolved=False
```

Comment:
I'm a bit stuck then.  I feel like I need your signoff that this runs so we can use it for Run 1.1.  Since it's a bit of the wild west with the obs packages, runability is the best test.

If I build a current stack for you on SLAC machines, would that make things better?